Find fontconfig using pkg-config

We already verify that it can be found like that during configure

From: Allan Sandfeld Jensen
Pick-to: 94-based 90-based
Task-number: QTBUG-61158
Change-Id: I569590e96a490c4ed6e6dc560fbd110d86d77956
Reviewed-by: Michal Klocek <michal.klocek@qt.io>


---
diff --git a/src/3rdparty/chromium/components/services/font/BUILD.gn b/chromium/components/services/font/BUILD.gn
index b065131e035..f3e544ed29f 100644
--- a/src/3rdparty/chromium/components/services/font/BUILD.gn
+++ b/src/3rdparty/chromium/components/services/font/BUILD.gn
@@ -42,6 +42,7 @@ if ((is_linux || is_chromeos) && enable_plugins) {
       "//base:base",
       "//ppapi/buildflags:buildflags",
       "//ppapi/c",
+      "//third_party/fontconfig",
     ]
   }
 }
diff --git a/src/3rdparty/chromium/third_party/fontconfig/BUILD.gn b/chromium/third_party/fontconfig/BUILD.gn
index 1bee7acb6b9..4cc32d62774 100644
--- a/src/3rdparty/chromium/third_party/fontconfig/BUILD.gn
+++ b/src/3rdparty/chromium/third_party/fontconfig/BUILD.gn
@@ -3,6 +3,7 @@
 # found in the LICENSE file.
 
 import("//build/config/sanitizers/sanitizers.gni")
+import("//build/config/linux/pkg_config.gni")
 import("//third_party/fontconfig/fontconfig.gni")
 
 assert(is_linux || is_chromeos)
@@ -110,8 +111,8 @@ if (use_bundled_fontconfig) {
     }
   }
 } else {
-  config("fontconfig_config") {
-    libs = [ "fontconfig" ]
+  pkg_config("fontconfig_config") {
+    packages = [ "fontconfig" ]
   }
 
   group("fontconfig") {
